cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,130 @@
+                        }
+                    },
+                },
+                "GetInstructor": {
+                    "response": {
+                        "data": {
+                            instructor
+                        },
+                    },
+                    "test": ({userID}) => {
+                        expect(userID).equals(instructor.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/instructor/${instructor.user.id}`);
+        });
+    });
+
+    it("Loads data properly into the first form", () => {
+        cy.fixture("users.json").then(({instructor}) => {
+            cy.get("[data-cy=basicInfo-firstName-input]")
+                .should("have.value", instructor.user.firstName);
+            cy.get("[data-cy=basicInfo-address-input]")
+                .should("have.value", instructor.address);
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .should("have.value", instructor.phoneNumber);
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.enabled");
+        });
+    });
+
+    it("Can enter data properly into the first form", () => {
+        cy.fixture("users.json").then(({instructor2}) => {
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .clear()
+                .fastType("0");
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.disabled");
+            cy.get("[data-cy=basicInfo-phoneNumber-input]").clear();
+            cy.get("[data-cy=basicInfo-phoneNumber-input]").fastType(instructor2.phoneNumber);
+            cy.get("[data-cy=basicInfo-address-input]")
+                .clear()
+                .fastType(instructor2.address)
+                .should("have.value", instructor2.address)
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.enabled");
+        }); 
+    });
+
+    it("Properly goes to the next part of the form and checks for data to load properly on the second form", () => {
+        cy.fixture("users.json").then(({instructor}) => {
+            cy.get("[data-cy=basicInfo-nextButton]").click();
+            cy.get("[data-cy=experience-subjects]")
+                .children()
+                .find("span")
+                .first()
+                .should("have.text", instructor.subjects[0].name);
+            cy.get("[data-cy=experience-experience-input")
+                .should("have.value", instructor.experience);
+            cy.get("[data-cy=experience-biography-input]")
+                .should("have.value", instructor.biography);
+            cy.get("[data-cy=experience-language-input]")
+                .should("have.value", instructor.language);
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.enabled");
+        });
+    });
+
+    it("Can enter data properly into the second form and go back to and from the first form with all data intact", () => {
```

Comment:
This test name is confusing. What you're doing in the code is:
1. Go to the previous form section
2. Check if the next button is still enabled, click if it is (you're not validating whether or not this previous section has its fields filled out)
3. Re-populate the second section's fields. 

Split this up into 2 tests:
1. it can go back to first instructor form section (use the specific title of the section instead of "first instructor form section")
  - Test to see if you can go back, and the info that you edited is still in its edited state
2. it fills out the second instructor form section (use the specific title of the section instead of "second instructor form section")